app/lib/get-user-classification-counts.js

```diff
@@ -0,0 +1,26 @@
+import getAllProjectPreferences from './get-all-project-preferences';
+import apiClient from 'panoptes-client/lib/api-client';
+
+function getUserClassificationCounts(user) {
+  return getAllProjectPreferences(user).then((projectPreferences) => {
+    const projectIDs = projectPreferences.map((projectPreference) => {
+      return projectPreference.links.project;
+    });
+    return apiClient.type('projects').get({ id: projectIDs, cards: true, page_size: projectPreferences.length }).catch(() => {
```

Comment:
Alright. I took out any projectPreference with an activity count of one or less.